bridge/client/app/_models/trace.ts

```diff
@@ -170,6 +170,15 @@ class Trace {
     return this.data.service;
   }
 
+  getChartLabel(): string {
+    return this.data.labels?.get("buildId") ?? this.time;
+    if(this.data.labels)
+      label = this.data.labels.get("buildId");
+    if(!label)
+      label = this.time;
+    return label;
```

Comment:
```suggestion
```
I'm sorry, this should also have been replaced, not just the single line, I still need to get used to Github. :smiley:

Reply:
Of course, I removed the unused code.